client/src/retry.rs

```diff
@@ -529,6 +531,16 @@ mod tests {
     use std::{ops::Add, time::Instant};
     use tonic::Status;
 
+    /// Predefined retry configs with low durations to make unit tests faster
+    const TEST_RETRY_CONFIG: RetryConfig = RetryConfig {
```

Comment:
This should be guarded by `#[cfg(test)]` so we make sure it's never accidentally used in real code

Reply:
isn't the whole module under `#[cfg(test)]` so it can't be used?